stdlib/sql/to.go

```diff
@@ -174,7 +174,8 @@ func (t *ToSQLTransformation) Process(id execute.DatasetID, tbl flux.Table) (err
 			return nil
 		}
 	}
-	return err
+
+	return t.db.Close()
```

Comment:
we probably want to keep this open for the duration of the transformation.  `Finish` is the right place.